src/lib.rs

```diff
@@ -74,8 +74,6 @@ assert_eq!(a, b.to_csc());
 
 */
 
-#![deny(warnings)]
```

Comment:
I've been wondering about removing this for a while, but I'm yet unsure. I've found this switch quite helpful to force me to think about warnings.

I suspect the rationale is that it could break some downstream crates with some compiler changes, which is a very valid concern, I'm wondering if it's possible to have this switch on only for dev builds?

Reply:
You can run CI with "RUSTFLAGS=-D warnings" and remove that line. Any PRs with warning should then fail

Reply:
OK. I put #![deny(warnings)] back, while I personally tend to use `RUSTFLAGS=-D warnings` as @mulimoen suggested. Anyway, it's up to you to decide. :)

Reply:
Nice to know, thanks @mulimoen